src/dialogxml/dialogs/dialog.cpp

```diff
@@ -452,6 +445,12 @@ void cDialog::recalcRect(){
 
 	winRect.right *= get_ui_scale();
 	winRect.bottom *= get_ui_scale();
+	// Uncomment if you need to measure any dialogs.
+	/*
+	LOG_VALUE(fname);
+	LOG_VALUE(winRect.right);
+	LOG_VALUE(winRect.bottom);
+	*/
```

Comment:
```suggestion
	//*/
```

Reply:
Is this a clever way to make it so the first `/*` can be removed without changing another line? Cool.